src/components/sbb-timetable-occupancy/sbb-timetable-occupancy.scss

```diff
@@ -19,28 +21,24 @@
   gap: var(--sbb-spacing-fixed-2x);
 }
 
-.occupancy__list-item {
+.sbb-timetable-occupancy__list-item {
   display: flex;
   align-items: center;
   flex-wrap: nowrap;
-  gap: #{sbb.px-to-rem-build(2)};
+  gap: var(--sbb-spacing-fixed-1x);
 }
 
-.occupancy__class--visual {
+.sbb-timetable-occupancy__list-item-class {
   @include sbb.text-s--regular;
-}
 
-.occupancy__class--visually-hidden {
-  @include sbb.screen-reader-only;
+  color: var(--sbb-timetable-occupancy-color);
+  line-height: 1rem;
```

Comment:
I took it from sbb-timetable-row.scss line 11 / 117 and there it's fixed to 16px (using pxToRemBuild)

Reply:
Could it maybe be just 1em to align to the current font-size? or line-height: 0? "1rem" a little bit looks random to me :D

Reply:
A deeper check has been done with @federicoisepponfincons: the rule is useful only for the sbb-timetable-row.

The three elements in the footer took the same line-height from the parent; now this is not possible, because of the new occupancy component.

should i create a variable in the occupancy and set it in the row component?
like:

```
// sbb-occupancy
line-height: var(--sbb-occupancy-line-height);

// sbb-row
:host(
  --sbb-occupancy-line-height: 1rem;
)
```

Reply:
solved with line-height: 1 and flex alignment in timetable-row